app/views/govuk_publishing_components/components/_share_links.html.erb

```diff
@@ -0,0 +1,36 @@
+<%
+  title ||= 'Share this page'
+  facebook_href ||= false
+  twitter_href ||= false
+%>
+<% if facebook_href || twitter_href %>
+  <div class="gem-c-share-links dont-print" data-module="track-share-button-clicks">
```

Comment:
The [corresponding Javascript](https://github.com/alphagov/government-frontend/blob/591807f8b0f477fa9e852b69fdce6ab736f32c81/app/assets/javascripts/modules/track-share-button-clicks.js) needs to be copied over for this tracking to work

Reply:
Ah, thanks, good spot. Is there a reason this isn't just using the standard tracking?

Reply:
I'm not sure, it looks like it was copied over when the links were moved from Whitehall, so it might just be it wasn't considered then? https://github.com/alphagov/government-frontend/pull/207

Reply:
Okay, I'll see if I can reproduce it using the standard module. Also if we pass the tracking, that makes it more flexible for the changes I'm planning.

Reply:
On second thought, I'll do that when I iterate the component, as it's a fairly major change and it'll block this PR. Have added the necessary JS - happy to approve?